template/lambda-sqs-worker-cdk/tsconfig.json

```diff
@@ -1,12 +1,12 @@
 {
   "compilerOptions": {
     "baseUrl": ".",
-    "lib": ["ES2020"],
+    "lib": ["ES2022"],
```

Comment:
Should we delete these `tsconfig.json`s and revert to `template/base/tsconfig.json`? Maybe it's better to keep these separate since we'll have the same window of divergence when Node.js 20 goes LTS.

Reply:
Yeah I had the same thought here too. Given Lambda and the ecosystem is always a little on the slow side to update I think it might be safer to keep it separate. Them updating this quick could have been a fluke :P